templates/medical-bill/templates/main.tpl

```diff
@@ -0,0 +1,282 @@
+    )}}
+
+    <table-cell>
+        <table columns="2" column-widths="0.8 0.2">
+            {{template "table-cell-paragraph" (extendDict $props "BackgroundColor" "#0261AB" "Margin" "2" "TextColor" "#ffffff" "Font" "helvetica-bold" "Text" "Current Balance" ) }}
+            {{template "table-cell-paragraph" (extendDict $props "BackgroundColor" "#0261AB" "Margin" "2" "Align" "right" "TextAlign" "right" "TextColor" "#ffffff" "Font" "helvetica-bold" "Text" .bill.Total ) }}
+
+            {{template "table-cell-paragraph" (extendDict $props "Colspan" 2 "Margin" "0" "BackgroundColor" "#ffffff" "TextColor" "#0261AB" "Align" "left" "TextAlign" "left"
+                "Text" "This is your first notice for the visit above, which includes a list of itemized services rendered.")}}
+
+            {{template "table-cell-paragraph" (extendDict $props "Colspan" 2 "Margin" "0" "TextColor" "#000000" "Font" "helvetica"
+                "Text" "We offer a Financial Aid program for qualified applicants. For more information, please call 123-456-7890 or visit our website at www.ourwebsite.com for more information.")}}
+        </table>
+    </table-cell>
+</table>
+
+<table columns="2" column-widths="0.5 0.5" margin="15 0 0 0">
+    {{template "table-cell-paragraph" (extendDict $props "Colspan" 2 "Align" "center" "TextAlign" "center" "Font" "helvetica-bold" "FontSize" 12 "Text" "Please retain statement for your records" ) }}
+
+    <table-cell colspan="2">
+        <line position="relative" fit-mode="fill-width" thickness="1" style="dashed" dash-array="10"></line>
+    </table-cell>
+
+    {{template "table-cell-paragraph" (extendDict $props "Colspan" 1 "Align" "left" "TextAlign" "left" "Font" "helvetica" 
+        "FontSize" 9 "Indent" 5 
+        "Text" "Please check box if address is incorrect or insurance information has changed, and indicate change(s) on reverse side.") }}
+
+    {{template "table-cell-paragraph" (extendDict $props "Rowspan" 2 "Align" "left" "TextAlign" "left" "VerticalAlign" "top" "Font" "helvetica-bold" 
+        "FontSize" 9 "Text" "IF PAYING BY VISA, MASTERCARD, DISCOVER OR AMEX, FILL OUT BELOW") }}
+
+    <table-cell>
+        <line position="relative" fit-mode="fill-width" thickness="1" margin="5 0"></line>
+    </table-cell>
+
+    <table-cell>
+        <division>
+            {{template "simple-paragraph" dict "Margin" "0" "LineHeight" 1 "Font" "helvetica-bold" "FontSize" 10 "TextColor" "#000000" "Text" "MAKE CHECKS PAYABLE TO"}}
+            {{template "simple-paragraph" dict "Margin" "0" "LineHeight" 1 "Font" "helvetica-bold" "FontSize" 10 "TextColor" "#0261AB" "Text" (printf "%s\n%s\n%s" .institution.name .institution.address1 .institution.address2)}}
+        </division>
+    </table-cell>
+
+    <table-cell rowspan="2">
+        <table columns="4" column-widths="0.25 0.25 0.25 0.25">
```

Comment:
No need to specify the `column-widths` attribute if you want all columns to be equal in width.